README.md

```diff
@@ -228,6 +228,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
 - `DangerousCollapseKeysUsage`: Disallow usage of `EntryStream#collapseKeys()`.
 - `JooqBatchWithoutBindArgs`: jOOQ batch methods that execute without bind args can cause performance problems.
 - `InvocationTargetExceptionGetTargetException`: InvocationTargetException.getTargetException() predates the general-purpose exception chaining facility. The Throwable.getCause() method is now the preferred means of obtaining this information. [(source)](https://docs.oracle.com/en/java/javase/17/docs/api//java.base/java/lang/reflect/InvocationTargetException.html#getTargetException())
+- `PreferInputStreamTransferTo`: Prefer JDK `InputStream.transferTo(OutputStream)` over utility methods such as `com.google.common.io.ByteStreams.copy(InputStream, OutputStream)`, `org.apache.commons.io.IOUtils.copy(InputStream, OutputStream)`, `org.apache.commons.io.IOUtils.copyLong(InputStream, OutputStream)`.
```

Comment:
I had to try really hard not to sort & clean this up in this PR, but I would like to create a separate PR to sort these and ensure we actually have coverage for all the current checks in baseline.

Reply:
I toyed with the upstream docs generator to programmatically generate this list, but it added more complexity to the build than I’d really like.

Reply:
We could throw together an annotation processor that just generates a `checks.md` similar to metric-schema. I might try that if I have some coding time during break.